docs/source/admin/runbook.rst

```diff
@@ -0,0 +1,255 @@
+
+
+Ambari Alerts
+^^^^^^^^^^^^^^
+
+Ambari uses the service indicators to convey the status of a service, when issues arise you can view detailed information on the Alerts page.
+
+.. image:: /_static/Runbook/image3.png
+
+A user can also set up email or slack alerting that is customizable.
+
+.. image:: /_static/Runbook/image4.png
+
+
+Backup and Recovery
+^^^^^^^^^^^^^^^^^^^
+Koverse relies on Accumulo for data storage, PostgreSQL for metadata storage, and a set of configuration files. A production backup strategy must incorporate all three. Here are some suggestions for each.
+
+**PostgreSQL**
+
+Use the `pg_dump <http://www.postgresql.org/docs/9.1/static/backup-dump.html>`_ command. To restore, simply re-create the postgres database from the backup.
+
+**Accumulo**
+
+Use the `Accumulo Export Tables <http://accumulo.apache.org/1.6/examples/export.html>`_ feature to backup the "kv_*" tables.
+
+**Configuration Files**
+
+Copy the entire koverse-server directory - specifically the /conf directory must be included.
+
+
+Distributed System Operations
+-----------------------------
+
+Koverse sits on top of a complex set of interworking and distributed services.
+These include:
+
+* Hadoop Distributed File System (HDFS)
+* Hadoop YARN
+* Spark
+* ZooKeeper
+* Accumulo
+* PostgreSQL
+
+
+Total System Startup
+^^^^^^^^^^^^^^^^^^^^
+
+There is an order to which the underlying systems should be brought online.
+When systems do not depend on each other they can be started at the same time.
+
+1. Data Storage and Coordination Layer - these can be started first after system boot.
+	* HDFS DataNodes
+	* HDFS NameNode
+	* ZooKeeper
+	* PostgreSQL
+
+2. Data Services Layer - all of these depend on one or more processes in the Storage and Coordination Layer.
+	* YARN ResourceManager
+	* YARN NodeManagers
+	* Accumulo Tablet Servers
+	* Accumulo Master
+
+3. Application Layer - all of these depend on one or more process in the Data Services Layer
+	* Accumulo Monitor
+	* Accumulo Garbage Collector
+	* Koverse Server
+	* Koverse Web App
+
+
+Total System Shutdown
+^^^^^^^^^^^^^^^^^^^^^
+Processes should be stopped in reverse of the startup layer order.
+
+1. Application Layer
+	* Koverse Server
+	* Koverse Web App
+	* Accumulo Monitor
+	* Accumulo Garbage Collector
+
+2. Data Services Layer
+	* YARN ResourceManager
+	* YARN NodeManagers
+	* Accumulo Tablet Servers
+	* Accumulo Master
+
+3. Data Storage and Coordination Layer
+	* HDFS DataNodes
+	* HDFS NameNode
+	* ZooKeeper
+	* PostgreSQL
+
+If a process in say, the Data Storage and Coordination Layer, is stopped before all processes in the Data Services and Application Layers, system state may become unstable or corrupt.
+All processes in one layer should be stopped before stopping any processes in the next layer.
+
+Sometimes a single worker process in a lower layer can be stopped and restarted without stopping higher layers.
+See "Fixing a simple, single-server failure".
+
```

Comment:
I like the way you grouped these into layers and integrated that into startup/shutdown.  Makes it much more intuitive for someone unfamiliar with the Hadoop stack.

The only suggestion I would make is that if we are including HDFS HA configurations in this, we might need to separate ZooKeeper and PostgreSQL into their own layer along with HDFS JournalNodes.  I believe that layer would need to start prior to a HA HDFS NameNode.

Reply:
I did not author this :) I just copied it from the opsguide.  I will create a layer for zk, postgres and the journal nodes. There is a ticket for HA, I can keep this in mind for those.

Reply:
cool, looks good